lib/elixir/scripts/elixir_docs.exs

```diff
@@ -56,7 +57,8 @@ canonical = System.fetch_env!("CANONICAL")
     "Getting started": ~r"pages/getting-started/.*\.md$",
     "Anti-patterns": ~r"pages/anti-patterns/.*\.md$",
     References: ~r"pages/references/.*\.md$",
-    "Meta-programming": ~r"pages/meta-programming/.*\.md$"
+    "Meta-programming": ~r"pages/meta-programming/.*\.md$",
+    Mix: ~r"pages/mix/.*\.md$"
```

Comment:
I believe I was referring to those pages as /mix-and-otp/. there should be existing links and warnings when you running `make docs_elixir`.